SupportClient/SupportClient/Controllers/ViewController.swift

```diff
@@ -0,0 +1,200 @@
+    
+    private(set) var countEditTaps: Int = 0
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = UIColor.backgroundColor
+        // Do any additional setup after loading the view.
+        
+        setupData()
+        setupNavigationBar()
+        setupFeedbackLabel()
+        setupConversationButton()
+        setupFeedbackTableView()
+        setupConstraints()
+    }
+    
+    // TODO: hook up to actual server to load the data
+    // This is dummy data for testing
+    // Add variables to feedbackData to show the tableView + editing functionality
+    func setupData() {
+        let jsonString = """
+        {
+            "title" : "Ithaca Transit Bug",
+            "message" : "This app sometimes glitches out on me and shows the wrong bus times",
+            "hasRead" : false
+        }
+        """
+        
+        guard let jsonData = jsonString.data(using: .utf8) else { return }
+        let feedback = try! JSONDecoder().decode(Feedback.self, from: jsonData)
+        
+        feedbackData = [feedback]
+    }
+    
+    func setupNavigationBar() {
+        navigationController?.navigationBar.barTintColor = UIColor.navigationTintColor
+        let attributes = [
+            NSAttributedString.Key.font: UIFont._21RobotoMedium!,
+            NSAttributedString.Key.foregroundColor: UIColor.titleColor
+        ]
+        let attributedTitle = NSAttributedString(string: "Feedback", attributes: attributes)
+        title = attributedTitle.string
+        navigationItem.rightBarButtonItem = UIBarButtonItem(
+            image: UIImage(named: "pen"),
+            style: UIBarButtonItem.Style.plain,
+            target: self,
+            action: #selector(handleNavigationBarRightTap)
+        )
+    }
+    
+    func setupFeedbackLabel() {
+        feedbackLabel.lineBreakMode = .byWordWrapping
+        feedbackLabel.numberOfLines = 0
+
+        // Format the alignment and spacing between lines
+        let paragraphStyle = NSMutableParagraphStyle()
+        paragraphStyle.alignment = .center
+        paragraphStyle.lineSpacing = 10
+        
+        let title = "No Feedback Yet\n"
+        let subtitle = "See feedback conversations here"
+        
+        let titleAttributes: [NSAttributedString.Key : Any] = [
+            NSAttributedString.Key.font: UIFont._17RobotoMedium!,
+            .paragraphStyle: paragraphStyle,
+            NSAttributedString.Key.foregroundColor: UIColor.titleColor
+        ]
+        let attributedText = NSMutableAttributedString(string: title, attributes: titleAttributes)
+        
+        let subtitleAttributes = [
+            NSAttributedString.Key.font: UIFont._13RobotoRegular!,
+            NSAttributedString.Key.foregroundColor: UIColor.subtitleColor
+        ]
+        attributedText.append(NSAttributedString(string: subtitle, attributes: subtitleAttributes))
+
+        feedbackLabel.attributedText = attributedText
+        feedbackLabel.sizeToFit()
```

Comment:
i dont think you need to call `sizeToFit()`

Reply:
Without `sizetoFit()` the label's height does not automatically adjust to fit the second line, so I need it.